styles.css

```diff
@@ -203,7 +203,7 @@ nav {
   display: grid;
   height: 386px;
   width: 327px;
-  background: linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0.2) 90.94%), url(assets/Img\ Placeholder2.svg);
+  background: url(assets/Img\ Placeholder2.svg);
```

Comment:
- [ OPTIONAL] I think it would be better to use % instead of px in order to make your website responsive. :recycle: